lib/item.ts

```diff
@@ -66,11 +66,12 @@ export class Item {
 	}
 
 	get trueProductionLimit(): number {
-		return Math.floor(
-			this.unitsOfProduction *
-				this.baseAmtProduced *
-				currentPerkMultiplier *
-				currentToolMultipliers[this.toolType],
+		return (
+			Math.floor(
+				this.unitsOfProduction *
+					currentPerkMultiplier *
+					currentToolMultipliers[this.toolType],
+			) * this.baseAmtProduced
```

Comment:
This is to apply the floor before getting the base amount to correct getting a little extra due to applying the floor late.